linera-service/src/linera/client_context.rs

```diff
@@ -138,12 +139,18 @@ impl ClientContext {
         Ok(Self::configure(options, wallet_state))
     }
 
-    pub fn wallet_state_mut(&mut self) -> &mut WalletState {
-        &mut self.wallet_state
+    /// Returns the `Wallet` as an immutable reference.
+    fn wallet(&self) -> &Wallet {
+        self.wallet_state.inner()
+    }
+
+    /// Returns the `Wallet` as a mutable reference.
```

Comment:
Nit: adding documentation links

```suggestion
    /// Returns the [`Wallet`] as an immutable reference.
    fn wallet(&self) -> &Wallet {
        self.wallet_state.inner()
    }

    /// Returns the [`Wallet`] as a mutable reference.
```